data/releases/eol.yaml

```diff
@@ -48,11 +48,10 @@ branches:
   release: "1.19"
 - endOfLifeDate: "2021-06-18"
   finalPatchRelease: 1.18.20
-  note: Created to solve regression introduced in 1.18.19
-  release: "1.18"
-- endOfLifeDate: "2021-05-12"
-  finalPatchRelease: 1.18.19
-  note: '[Regression](https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs)'
+  note: >-
```

Comment:
I'm not sure I understand this change, can you please clarify it?

Reply:
I'm also getting this diff when running schedule-builder:

```diff
diff --git a/data/releases/eol.yaml b/data/releases/eol.yaml
index 739b542b3f..e28b84f20a 100644
--- a/data/releases/eol.yaml
+++ b/data/releases/eol.yaml
@@ -48,9 +48,7 @@ branches:
   release: "1.19"
 - endOfLifeDate: "2021-06-18"
   finalPatchRelease: 1.18.20
-  note: >-
-    Release 1.18.20 was created to solve a
-    [regression](https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs)
+  note: Release 1.18.20 was created to solve a [regression](https://groups.google.com/g/kubernetes-dev/c/KuF8s2zueFs)
     introduced in 1.18.19
   release: "1.18"
 - endOfLifeDate: "2021-01-13"
 -
```

Reply:
Ah, this file is autogenerated? Would be great if we can include a comment about that.

Reply:
My intention making changes to this file was to fix some issues about how well the data fitted what the website needs - see https://github.com/kubernetes/website/pull/46368/commits/633460f92e3324f418e8561e2377e919c69af6fb

Reply:
Oops; when I started this PR, the file didn't have the helpful comment about being autogenerated.

Can we still fix v1.18 having two EOL dates?

Reply:
Yes, fixing that is fine, I also left a comment about fixing it for the v1.22 release.